docset/winserver2025-ps/storagereplica/New-SRGroup.md

```diff
@@ -141,12 +134,8 @@ Accept wildcard characters: False
 
 ### -EnableCompression
 
-Indicates that this Storage Replica group should use SMB compression for data transfer. This
-parameter only applies to
-[Azure Stack HCI version 22H2](https://azure.microsoft.com/updates/public-preview-azure-stack-hci-version-22h2/)
-or later, and Windows Server Datacenter: Azure Edition beginning with the 2022-09 Cumulative
-Update for Microsoft server operating system version 21H2 for x64-based Systems
-([KB5017381](https://support.microsoft.com/help/5017381)).
+Indicates that this Storage Replica group should use SMB compression for data transfer.
```

Comment:
add: 

SMB compression is only used with TCP, not SMB Direct and RDMA networks.